src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1911,29 +1911,22 @@ void OrbitMainWindow::AppendToCaptureLog(CaptureLogSeverity severity, absl::Dura
             severity_name);
 }
 
-void OrbitMainWindow::SetCallstackInspection(
-    std::shared_ptr<const CallTreeView> top_down_view,
-    std::shared_ptr<const CallTreeView> bottom_up_view,
-    orbit_data_views::DataView* callstack_data_view,
-    const orbit_client_data::CallstackData* callstack_data,
-    const orbit_client_data::PostProcessedSamplingData* post_processed_sampling_data) {
-  ui->topDownWidget->SetInspection(std::move(top_down_view));
-  ui->bottomUpWidget->SetInspection(std::move(bottom_up_view));
-
-  ui->samplingReport->hide();
-  ui->inspectionReport->SetInspection(app_.get(), callstack_data_view, callstack_data,
-                                      post_processed_sampling_data);
-  connect(ui->inspectionReport, &OrbitSamplingReport::LeaveCallstackInspectionClicked, this,
-          [this]() { app_->ClearInspection(); });
-  ui->inspectionReport->show();
-}
-
-void OrbitMainWindow::ClearCallstackInspection() {
-  ui->topDownWidget->ClearInspection();
-  ui->bottomUpWidget->ClearInspection();
-  ui->inspectionReport->hide();
-  ui->inspectionReport->Deinitialize();
-  ui->samplingReport->show();
+void OrbitMainWindow::SetSelection(SelectionData& selection_data) {
```

Comment:
It seems you're only accessing `const` member functions of `SelectionData`. So what about making the parameter a const-ref?

Reply:
I meant to :) thanks for catching!